hardhat.config.js

```diff
@@ -35,11 +35,18 @@ const optimizerSettings = {
     },
 };
 
+const outputSelectionSettings = {
```

Comment:
why is this needed? as per this [doc](https://www.npmjs.com/package/hardhat-storage-layout) only the require statement needs to be added? Please add a comment if it's workaround that's needed so it's clear in the future

Reply:
Without `outputSelection` option, `STORAGE_LAYOUT=true npx hardhat check` command from the [hardhat-storage-layout](https://www.npmjs.com/package/hardhat-storage-layout) plugin returns `Table has empty fields`. This is because there is no output generated for storage layout. With `outputSelection` option using hardhat, the command shows expected results of storage layout.

Reply:
If you see the plugin [code](https://www.npmjs.com/package/hardhat-storage-layout?activeTab=code) itsets up automatically `outputSelection option with storageLayout` while compile the contract with hardhat. But since we're using customized compilerSettings, it could not be applied.